js/menu.js

```diff
@@ -0,0 +1,25 @@
+const html = document.getElementsByTagName('html')[0];
+const navMenu = document.querySelector('.desktop-nav');
+const showBtn = document.querySelector('nav .menu-bars');
+const closedBtn = document.querySelector('nav .closed-menu-bars');
+const navLinks = document.querySelectorAll('nav ul li');
+
+function showMenu() {
+  navMenu.classList.add('open');
+  closedBtn.style.display = 'block';
+  html.style.overflow = 'hidden';
+}
+
+function closeMenu() {
+  navMenu.classList.remove('open');
+  closedBtn.style.display = 'nonne';
+  html.style.overflow = 'auto';
```

Comment:
When the user clicks on the close (X) button, the mobile menu disappears but the close button still appears. Kindly fix this as suggested by the previous reviewer. I think it would be better if you implemented the code provided.

NOTE: This is part of the project requirements, it is very important that you look into it.